sktime/transformations/series/detrend/_deseasonalize.py

```diff
@@ -175,16 +199,56 @@ def update(self, Z, X=None, update_params=False):
 class ConditionalDeseasonalizer(Deseasonalizer):
     """Remove seasonal components from time series, conditional on seasonality test.
 
+    Fit tests for seasonality and if the passed time series has a seasonal component
+    it applies seasonal decomposition provided by statsmodels to compute
+    the seasonal component. If the test is negative `_seasonal` is set to all ones
```

Comment:
Same as in Deseasonalizer -- is reference to \_seasonal really a reference to seasonal\_?